frontend/src/components/App.tsx

```diff
@@ -21,45 +24,47 @@ interface AppState {
 }
 
 class App extends React.Component<AppProps, AppState> {
-
+  
   constructor(props) {
     super(props);
     
     this.state = {
       navigationDrawerOpen: false
-    };
+    };  
   }
 
-  // Handles opening drawer in AppBar and Drawer components
+  // Handles opening drawer in AppBar and Drawer components 
   handleDrawerOpen = () => {
     this.setState({navigationDrawerOpen: true});
   };
 
-  // Handles closing drawer in AppBar and Drawer components
+  // Handles closing drawer in AppBar and Drawer components 
```

Comment:
```suggestion
  // Handles closing drawer in AppBar and Drawer components
```
Please remove this whitespace (see https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace).